test/utils.py

```diff
@@ -58,3 +59,11 @@ def mocked_count_method(*_):
     Simulates a count query
     """
     return COUNT_SAMPLE_MAX
+
+
+def debug_subprocess_pytest(result):
```

Comment:
used only in test -> move it to test folder

Reply:
this is already on the test folder